fs/xfstests.py

```diff
@@ -43,17 +43,20 @@ def setUp(self):
         detected_distro = distro.detect()
 
         packages = ['e2fsprogs', 'automake', 'gcc', 'quota', 'attr',
-                    'make',  'xfsprogs',  'gawk', 'fio', 'dbench']
+                    'make',  'xfsprogs',  'gawk']
 
         if 'Ubuntu' in detected_distro.name:
             packages.extend(['xfslibs-dev', 'uuid-dev', 'libtool-bin', 'libuuid1',
-                             'libattr1-dev', 'libacl1-dev', 'libgdbm-dev', 'uuid-runtime'])
+                             'libattr1-dev', 'libacl1-dev', 'libgdbm-dev',
+                             'uuid-runtime', 'libaio-dev', 'fio', 'dbench'])
 
         elif detected_distro.name in ['centos', 'fedora', 'redhat']:
             packages.extend(['acl', 'bc', 'dump', 'indent', 'libtool', 'lvm2',
-                             'xfsdump', 'psmisc', 'sed', 'libacl-devel', 'libattr-devel',
-                             'libaio-devel', 'libuuid-devel', 'openssl-devel', 'xfsprogs-devel',
-                             'btrfs-progs-devel'])
+                             'xfsdump', 'psmisc', 'sed', 'libacl-devel',
+                             'libattr-devel', 'libaio-devel', 'libuuid-devel',
+                             'openssl-devel', 'xfsprogs-devel', 'btrfs-progs-devel'])
+            if detected_distro.name != 'redhat':
```

Comment:
Wrong indentation here. This `if` condition is under `Ubuntu`. I guess you wanted `redhat` condition parallel to `Ubuntu` not under `Ubuntu`.